tests/e2e/util.go

```diff
@@ -2557,3 +2555,16 @@ func expectedAnnotation(ctx context.Context, client clientset.Interface, pvclaim
 	})
 	return waitErr
 }
+
+//getRestConfigClient returns  rest config client
+func getRestConfigClient() *rest.Config {
+	// Get restConfig
+	var restConfig *rest.Config
```

Comment:
Move this after line 81, and check here is `restConfig` in `nil`, if `nil` continue with creating a client otherwise return `restConfig`

Reply:
done